app.py

```diff
@@ -92,15 +130,46 @@ def submit_timelog_form(ack, respond, body, logger):
 @app.command("/help")
 def help(ack, respond, command):
     ack()
-    respond()
+    output = """
+    \n*User Commands:*
+    */timelog* Opens a time logging form
+    */deletelast* Delete your last entry
+    */myentries n* Get a table with your last n entries
+    \n*Admin Commands:*
```

Comment:
Is it possible to only show these admin commands if the user that ran `/help` is an admin?